### ISSUE TITLE
docs: we need an UPGRADING.md file

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq#936]
Let's document the standard steps which should be done when ever you do a "git pull" from the latest version.

This should include things like:
 
1. check the CHANGELOG, check the NEWS file
2. do a git pull (how to do this properly?)
3. this guide should be linked from the User Guide as well as from the *main page*
4. Always check the DB changes in 
- ES
- Postgresql
- Mongo
- etc
5. Reloading & restarting the "botnet".

Basically this should be easily readable for non-intelmq devs / users. Think: ops-team.

**Comment by wagner-certat** [comment]
For reference the existing docs: https://github.com/certtools/intelmq/blob/master/docs/User-Guide.md#upgrade

**Comment by aaronkaplan** [comment]
Yes, as discussed right now - an ops-team will not want to read the full docs.
They need a separate (we can split the User Guide) UPGRADING.md document.

**Comment by wagner-certat** [comment]
The User-guide currently holds sections for installation, configuration, upgrading, utilities, uninstall, FAQ and a very short Additional Information.

If we separate the upgrade instructions, should we do the same for installation too? And probably other sections?
